cmd/emulator/Dockerfile

```
@@ -15,10 +15,11 @@ RUN git config --global 'url.ssh://git@github.com/.insteadOf' https://github.com
 
 COPY . .
 
+ARG TARGETOS TARGETARCH
```

I believe this should be declared separately as:
```suggestion
ARG TARGETOS 
ARG TARGETARCH
```
The way it is now I get an error when building and also according to reference you can't do that https://docs.docker.com/engine/reference/builder/#arg

---

Thanks - adjusted as you mentioned.
Locally I can confirm this now works:
```
docker buildx create --use
# ^ should only need to be run once
docker buildx build --platform=linux/amd64,linux/arm64 . -f cmd/emulator/Dockerfile
`
Reference:
https://www.docker.com/blog/faster-multi-platform-builds-dockerfile-cross-compilation-guide/